requirements.txt

```diff
@@ -1,8 +1,10 @@
 certifi==2018.1.18
 chardet==3.0.4
+humanfriendly==4.8
 idna==2.6
+prompt-toolkit==1.0.15
 requests==2.18.4
+six==1.11.0
 urllib3==1.22
-argparse==1.4.0
-humanfriendly==4.8
-prompt-toolkit==1.0.15
+wcwidth==0.1.7
+websockets==4.0.1
```

Comment:
We don't use the "websockets" module.

Reply:
Whoops. The `websockets` library is part of my local virtualenv which is left over from trying out the Slack API Real Time Messaging.